deploy/docs/Best_Practices.md

```diff
@@ -255,6 +255,34 @@ See the following links to official Fluentd buffer documentation:
 - https://docs.fluentd.org/configuration/buffer-section
 - https://docs.fluentd.org/buffer/file
 
+### Fluentd buffer size for metrics
+
+Should you have any connectivity problems, depending on the buffer size your setup will be able to survive for a given amount of time without a data loss,
+delivering the data later when everything is operational again.
+
+To calculate this time you need to know how much data you send. For the calculations below we made an assumption that a single metric data poing is around 1
+kilobyte in size, including metadata. This assumption is based on the average data we ingest. By default, for file based buffering we use gzip compression
+which gives us around 3:1 compress ratio.
+
+That results in `1 DPM` (Data Points per Minute) using around `333 bytes of buffer`. That is `333 kilobytes for 1 thousand DPM` and `333 megabytes for 1 million DPM`.
```

Comment:
Perhaps mention explicitly that this is storage used per minute.

Reply:
But that is already in minutes as DPM is in minutes 🤔 I'm not sure how to make that more clear.